views/components/select-token-modal/token-modal-item.tsx

```diff
@@ -34,16 +34,16 @@ const TokenModalItem: FC<TokenModalItemProps> = ({
 
   return (
     <Box
-      p="1rem"
+      p="xl"
       display="flex"
       color="textSoft"
       cursor="pointer"
       alignItems="center"
-      nHover={{ bg: 'rgba(0, 83, 219, 0.08)' }}
       justifyContent="space-between"
-      bg={selected ? 'rgba(0, 83, 219, 0.08)' : 'unset'}
       onClick={selected ? undefined : onClick}
       transition="background 500ms ease-in-out"
+      nHover={{ bg: 'rgba(0, 83, 219, 0.08)' }}
+      bg={selected ? 'rgba(0, 83, 219, 0.08)' : 'unset'}
```

Comment:
try see if this colors dosent exist on theme

Reply:
there is not alpha colors